addon/templates/components/power-select/options.hbs

```diff
@@ -5,18 +5,29 @@
 {{/if}}
 {{#each options as |opt index|}}
   {{#if (ember-power-select-is-group opt)}}
-    <li class="ember-power-select-group" aria-disabled={{ember-power-select-true-string-if-present opt.disabled}} role="option">
-      <span class="ember-power-select-group-name">{{opt.groupName}}</span>
-      {{#component optionsComponent
-        options=(readonly opt.options)
-        select=(readonly select)
-        groupIndex=(concat groupIndex index ".")
-        optionsComponent=(readonly optionsComponent)
-        role="group"
-        class="ember-power-select-options" as |option|}}
-        {{yield option select}}
-      {{/component}}
-    </li>
+    {{#with
```

Comment:
No problem...

I decided to keep them separate so you wouldn't have to worry about instantiating the options component in the group component when overriding the group component. All you have to worry about is adding a `yield` when overriding the group component, and this structure handles rendering the options component in place.